ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/visitor/infraestructure/antlr/visitors/classes/JavaConstructorVisitor.java

```diff
@@ -0,0 +1,73 @@
+package edu.usb.argos.ASTProcessor.visitor.infraestructure.antlr.visitors.classes;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import edu.usb.argos.ASTProcessor.antlr.JavaParser;
+import edu.usb.argos.ASTProcessor.antlr.JavaParserBaseVisitor;
+import edu.usb.argos.ASTProcessor.visitor.core.entities.classes.ConstructorInformation;
+import edu.usb.argos.ASTProcessor.visitor.core.interfaces.visitor.IConstructorAnalyzer;
+
+public class JavaConstructorVisitor extends JavaParserBaseVisitor<List<ConstructorInformation<JavaParser.BlockStatementContext>>> implements IConstructorAnalyzer<JavaParser.ClassBodyContext, JavaParser.BlockStatementContext> {
+
+    @Override
+    public List<ConstructorInformation<JavaParser.BlockStatementContext>> visitConstructors(JavaParser.ClassBodyContext classContext) {
```

Comment:
refactor this method, try not to exceed 20 lines of code

Reply:
Done